### PR TITLE
Added jquery.dotdotdot

### PR DESCRIPTION
**Pull request by gig177**

### CODE REVIEW

File: package-overrides/github/BeSite/jQuery.dotdotdot@1.7.4.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "main": "jquery.dotdotdot",
+    "deps": [
```
Comment (guybedford) [inline package-overrides/github/BeSite/jQuery.dotdotdot@1.7.4.json]: This should be a `dependenices` object. See other overrides for examples.

**Comment by gig177** [comment]
How should i specify install jquery too?

**Comment by guybedford** [comment]
See for example https://github.com/jspm/registry/commit/a3efb64471c6a1a1c3a01ec1a1f1003b5b86b60c#diff-50bfc48565a6e99b030a3cafe14adecaR18.

**Comment by gig177** [comment]
Thanks.

File: package-overrides/github/BeSite/jQuery.dotdotdot@1.7.4.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        "jquery": "*"
+    },
+    "shim": {
+        "jquery.dotdotdot": {
+            "deps": [
+                "jquery"
+            ]
+        }
+    },
+    "directories": {
```
Comment (guybedford) [inline package-overrides/github/BeSite/jQuery.dotdotdot@1.7.4.json]: We don't encourage `directories.lib` as there is a chance a user may have already installed this and be relying on the existing paths. Best to include `src/js` in all paths for compatibility. The only time we can safely set `directories.lib` is if this is checked into the original package.json file as a PR and bumped into a major release.

**Pull request closed without merging**